src/packagedcode/dockerfileLABEL.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+
+import io
+from pathlib import Path
+from dockerfile_parse import DockerfileParser  
+from packagedcode import models
+from packagedcode import utils
+
+
+class DockerfileHandler(models.DatafileHandler):
+    datasource_id = 'dockerfile'

```

Comment:
```suggestion
    datasource_id = 'dockerfile_oci_labels'
```
Here I think more explicit is better, as this is not really about the usual dockerfile contents but only the OCI labels